playbooks/k8s-services.yml

```diff
@@ -24,6 +24,34 @@
   tags:
     - always
 
+# Install OpenShift
+- hosts: kube-master
+  become: true
+  tasks:
+    - name: install virtualenv
+      apt:
+        name: 
+          - "virtualenv"
+        state: present
+      when: ansible_distribution == 'Ubuntu'
+    - name: create location for python virtual env
+      file:
+        path: "{{ deepops_dir }}"
```

Comment:
These vars need to be defined somewhere. Probably this just needs to be a role(s) but it's ok to just add a vars segment in here for now

Reply:
Yeah, agreed on this being a role. I'll just stick with defining the vars in place for now.